core/model/src/main/java/org/activityinfo/model/formTree/FormClassProviders.java

```diff
@@ -0,0 +1,28 @@
+package org.activityinfo.model.formTree;
+
+import org.activityinfo.model.form.FormClass;
+import org.activityinfo.model.resource.ResourceId;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class FormClassProviders {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)